src/vsg/platform/win32/Win32_Window.cpp

```diff
@@ -122,7 +122,7 @@ KeyboardMap::KeyboardMap()
             {VK_RETURN                            ,              KEY_Return},
             {VK_SHIFT                             ,              KEY_Undefined},
             {VK_CONTROL                           ,              KEY_Undefined},
-            {VK_MENU                              ,              KEY_Undefined},
+            {VK_MENU                              ,              KEY_Alt_L},
```

Comment:
Why wouldn't this many to KEY_Menu?

Reply:
[VK_MENU on Windows is the (left) 'Alt' key](https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes). KEY_Menu is (according to [KeyEvent.h](https://github.com/vsg-dev/VulkanSceneGraph/blob/2d238246a09c3af1d4f268e02bc232442c4c0156/include/vsg/ui/KeyEvent.h#L161)) supposed to be "VK_APPS, the context-menu key"